api

```diff
@@ -693,17 +693,17 @@ flatpak_install() { #install an app using flatpak
     buster)
       debian_ppa_installer "theofficialgman/flatpak-no-bwrap" "bionic" "0ACACB5D1E74E484"
       apt_lock_wait
-      sudo apt-get --only-upgrade install flatpak -y
+      sudo apt --only-upgrade install flatpak -y
```

Comment:
I generally use apt-get when I can since it avoids the warning in logs that apt is not a stable api

Reply:
Resolved in https://github.com/Botspot/pi-apps/pull/2286/commits/2e0bfe556fcb4513da7e20942f68b6fe8e144b0d